### ISSUE TITLE
C# Interactive, Roslyn & .NET Version

### ISSUE THREAD

**Issue by series0ne** [dotnet/roslyn#12924]
**Scenario**
I'm writing a .NET Core application targetting netcoreapp1.0/dnxcore50

**Problem**
The .NET target in the C# Interactive window does not reflect the target of the current project.

**Example**
In the C# Interactive window `typeof(Guid).GetProperties();`  compiles.
In my .NET Core app, `typeof(Guid).GetProperties();` causes a compiler error.

**Solution**
The .NET target in the C# Interactive window should reflect the current project or should be manually configurable through the C# Interactive toolbar

**Comment by tmat** [comment]
Correct. C# Interactive currently only supports targeting .NET Framework 4.6+

**Comment by alexzaytsev-newsroomly** [comment]
@tmat any plans to extend it to .net Core?

**Comment by ahdung** [comment]
And How toggle x86/x64?

**Comment by daverayment** [comment]
You can use the `#reset` command to switch between .NET Framework 32-bit and 64-bit, and the .NET Core environments. For instance:

`#reset core` will setup the window for Core.

I don't know why VS can't automatically do this when the project targets .NET Core, and it's a shame this suggestion is still open more than 6 years after initially being raised.
